2d-gettingstarted-shared-entities.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Getting Started - Adding Trackable Entities to an existing project
+home: true
+---
+
+This sums up the requirements a project has to comply to for **Trackable Entities** to work properly.
+
+## Declaration
+### 1. Server entities
+ - Implement `ITrackable`
+ - Implement `IMergeable`
+
+### 2. Client entities
+ - Inherit from `EntityBase`
+ - All collection reference properties must be an instance of `ChangeTrackingCollection` (CTC).* All navigation properties must internally wrap the navigated property in a singular CTC as demonstrated [here](https://github.com/TrackableEntities/trackable-entities/blob/develop/Samples/VS2013/WebApiSample.Inheritance/SampleWebApi.Inheritance.Shared.Entities/Models/Product.cs#L83)
+
+##Usage
```

Comment:
This Usage section doesn't belong here in this doc, because it describes general usage versus customizing entities.